samples/gke_dashboard_disable.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GKEDashboardConstraintV1
+metadata:
+  name: disable_gke_dashboard
+  annotations:
+    # Example of tying a constraint to a CIS benchmark
+    benchmark: GKE_HARDENING_GUIDELINE
```

Comment:
I thought CIS benchmark is organized by sections? What's the convention here?

Reply:
This is the GKE hardening guidelines, not part of CIS:
https://cloud.google.com/kubernetes-engine/docs/how-to/hardening-your-cluster

I don't know what's the best way of grouping them together hence the GKE_HARDENING_GUIDELINE as the benchmark.